test/podFailover/workload/scenarios/3pod3pvc.yaml

```diff
@@ -0,0 +1,22 @@
+storageClass:
+  name: azdisk-pod-failover-3pod3pvc-sc
+  parameters:
+    cachingmode: None
+    skuName: Premium_LRS
+    maxShares: "2"
+  additionalValues:
+    reclaimPolicy: Delete
+    volumeBindingMode: Immediate
+    allowVolumeExpansion: true
+  provisioner: disk.csi.azure.com
+
+namespace: azdisk-pod-failover-3pod3pvc
+
+podCount: 3
+
+pvcCount: 3
+
+failureType: delete-pod
```

Comment:
This scenario was originally being run with all the pods on the same node. I wonder if this should be changed to "same-node-failover" ?

Reply:
I have a same node failover template that I can make 3pod3pvc